source/mainnet/net/browser-wallet/setup-browser-wallet.rst

```diff
@@ -0,0 +1,72 @@
+.. include:: ../../variables.rst
+.. _setup-browser-wallet:
+
+=========================
+Set up the |bw|
+=========================
+
+The |bw| is a digital wallet that enables you to create and manage your Concordium
+:ref:`identities<glossary-identity>` and :ref:`accounts<glossary-account>` and to create transactions, such as sending CCD.
+
+To learn more about identities and accounts, see :ref:`identities<reference-id-accounts>` and :ref:`accounts<managing_accounts>`.
+
+Read the following guide to learn how to set up the wallet.
+
+Get started
+===========
+
+#. Download the |bw| extension. See :ref:`Downloads <downloads>`.
+
+#. Open a Google Chrome browser. Click **Extensions**. Select **Manage Extensions**.
+
+#. Turn on Developer mode. Click Load unpacked.
+
+#. Add the extension from the folder to which it was downloaded (and maybe unzipped, only at the dist file folder). You can click Extensions and "pin" an icon to the toolbar for the browser wallet if you want, or set up a keyboard shortcut for it. Otherwise, click Extensions whenever you want to start the browser wallet.
+
+#. Create a six-digit passcode or use a full password.
+
+#. Choose whether you are creating a new wallet or :ref:`recovering an existing one<recovery-wallet>`.
+
+   .. image:: ../images/browser-wallet/wallet-choice.png
+            :width: 25%
+
+Recovery phrase setup
+=====================
+
+If you are creating a new wallet, you must set up a recovery phrase. This is a 24 word phrase that stores your private keys, identities, and accounts. You must write down and confirm your recovery phrase. It is important to keep this secret recovery phrase in a safe location in case you need to recover your wallet on a new device.
+
+#. Once you click on **Create** you are shown the 24 word secret recovery phrase. Write it down and click **Continue**.
+
+   .. image:: ../images/browser-wallet/recovery-phrase.png
+            :width: 25%
+
+#. Enter all 24 words of your secret recovery phrase to confirm it. Click **Continue**.
+
+#. Choose whether to connect to Mainnet or Testnet to create your wallet.
+
+Request your identity
+=====================
+
+Having set up your passcode, you must then submit a request for an identity.
+
+#. If you don’t have any identities, click the ID cards drop-down.
```

Comment:
You can do this whether you have any identities or not.
And we have added a link on the page, if you don't have an identity, to the request page.

Reply:
A link on what page? I'm not clear what you mean.

Reply:
I think I meant that the page that would show you your identity, would have a button to go to the issuance page:
(But also account to account creation, and account creation to identity issuance (if there are no identities)

![Screenshot from 2022-09-21 09-30-47](https://user-images.githubusercontent.com/87635671/191442500-744442bd-d3e8-49d6-be47-f8307ba3f431.png)
![Screenshot from 2022-09-21 09-30-58](https://user-images.githubusercontent.com/87635671/191442498-9ba5a734-f7a1-4d16-b1c3-f5aad497b775.png)
![Screenshot from 2022-09-21 09-31-06](https://user-images.githubusercontent.com/87635671/191442492-e0124302-ad3d-4b0a-96a9-4da846712b7a.png)